components/LogWorkout/LogWorkout.jsx

```diff
@@ -27,6 +30,7 @@ const parseExercises = (exercises) => exercises.map((exercise) => {
   const exerciseType = exercise.sets[0].reps ? 'REPS' : 'SECS';
 
   return {
+    ID: exercise.exerciseId,
```

Comment:
```suggestion
    id: exercise.exerciseId,
```

Can we make this consistent with other code.